### PR TITLE
Host moc interpreter as part of the SDK site.

### PR DESCRIPTION
**Pull request by olaszakos**

### CODE REVIEW

**Review by chenyan-dfinity — APPROVED** [review]
LGTM. I think it's safe to remove the 0.5.14 version.

File: src/partials/footer-scripts.hbs [hunk 1]
```diff
@@ -51,6 +51,6 @@ function loadBase() {
   addPackage('base', 'dfinity/motoko-base', '{{page.attributes.moc-base-tag}}', 'src');
 }
 </script>
-<script async src="https://download.dfinity.systems/motoko/{{page.attributes.moc-version}}/js/moc-interpreter-{{page.attributes.moc-version}}.js" onload="loadBase()">
+<script async src="{{uiRootPath}}/js/moc/moc-interpreter-{{page.attributes.moc-version}}.js" onload="loadBase()">
```
Comment (chenyan-dfinity) [inline src/partials/footer-scripts.hbs:54]:
```suggestion
<script async src="{{uiRootPath}}/js/moc/moc-interpreter-0.6.11.js" onload="loadBase()">
```
As we copy the interpreter in this repo, the external `moc-version` attribute doesn't make much sense.

Comment (olaszakos) [reply src/partials/footer-scripts.hbs:54]: Thanks. Should we remove it from the playbook repo as well? I'm not sure if it's used anywhere else outside of this file. I don't have access to all the playbook modules. Same question about `moc-base-tag`.

Comment (chenyan-dfinity) [reply src/partials/footer-scripts.hbs]: Yes, it's safe to remove `moc-version` from the playbook. We want to keep the `moc-base-tag`, it's still in use.

**Comment by crusso** [comment]
@olaszakos is there any reason this hasn't been merged? I must admit I completely forgot about the interpreter update....

**Comment by olaszakos** [comment]
@crusso No reason, it got off my radar. Merging now.

**Pull request merged**